api/controllers/index.js

```diff
@@ -0,0 +1,3 @@
+const userController = require("./userController");
+
+module.exports = userController;
```

Comment:
추후에는 userController 뿐만 아니라 productController, cartController 등등 추가될 여지가 있기 때문에, exports도 객체 형태로 보내주는것이 용이해보입니다~!

route를 제외한 다른 layer들의 index.js도 마찬가지입니다! 모두 반영 부탁드릴게요 :)
```suggestion
const userController = require("./userController");

module.exports = { userController };
```